### PR TITLE
LPD-44387

### PR DESCRIPTION
**Pull request by gabrielwas**
https://liferay.atlassian.net/browse/LPD-44387

### CODE REVIEW

**Comment by gabrielwas** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
ci:test:relevant

File: modules/apps/object/object-api/src/main/java/com/liferay/object/petra/sql/dsl/DynamicObjectDefinitionTableUtil.java [hunk 1]
```diff
@@ -56,7 +56,14 @@ public static String getDataType(String businessType, String dbType) {
 		int size = 280;
 
 		if (StringUtil.equals(
-				businessType, ObjectFieldConstants.BUSINESS_TYPE_PICKLIST)) {
+				businessType,
+				ObjectFieldConstants.BUSINESS_TYPE_MULTISELECT_PICKLIST)) {
+
+			size = 5000;
```
Comment (carolmariaabb) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/petra/sql/dsl/DynamicObjectDefinitionTableUtil.java:62]: @gabrielwas why did you choose 5000?

File: modules/apps/object/object-api/src/test/java/com/liferay/object/petra/sql/dsl/DynamicObjectDefinitionTableUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	@Test
+	public void testGetDataType() {
+		Assert.assertEquals(
+			"BOOLEAN",
+			DynamicObjectDefinitionTableUtil.getDataType(
+				ObjectFieldConstants.BUSINESS_TYPE_BOOLEAN,
+				ObjectFieldConstants.DB_TYPE_BOOLEAN));
+
+		Assert.assertEquals(
+			"VARCHAR(5000)",
```
Comment (carolmariaabb) [inline modules/apps/object/object-api/src/test/java/com/liferay/object/petra/sql/dsl/DynamicObjectDefinitionTableUtilTest.java:35]: @gabrielwas why didn't you change it to `STRING` instead?

**Review by carolmariaabb — APPROVED** [review]

**Comment by gabrielwas** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
ci:forward:force

**Comment by gabrielwas** [comment]
ci:forward:force

**Pull request closed without merging**